apps/strapi-cms/src/api/webinar/content-types/webinar/schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "kind": "collectionType",
+  "collectionName": "webinars",
+  "info": {
+    "singularName": "webinar",
+    "pluralName": "webinars",
+    "displayName": "Webinar",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "title": {
+      "type": "string"
+    },
+    "cover_image": {
+      "type": "media",
+      "multiple": false,
+      "required": false,
+      "allowedTypes": [
+        "images",
+        "files",
+        "videos",
+        "audios"
+      ]
+    },
+    "description": {
+      "type": "blocks"
+    },
+    "slug": {
+      "type": "string"
+    },
+    "text_content": {
+      "type": "blocks"
+    },
+    "vimeo_video_id": {
```

Comment:
If we use the `id` the editor will be forced to always use Vimeo player and we need to hardcode the path in the FE. What about using the more generic `playerSrc`?